src/pcapi/admin/custom_views/mixins/suspension_mixin.py

```diff
@@ -66,6 +67,15 @@ def column_formatters(self):
     def user_list_url(self):
         return url_for(".index_view")
 
+    @action("unsuspend_selection", "Réactiver la sélection")
+    def action_bulk_edit(self, ids):
+        if not _allow_suspension_and_unsuspension(current_user):
+            return Forbidden()
+
+        users_api.bulk_unsuspend_account(ids, current_user)
+        flash("Les comptes utilisateurs ont été réactivés.")
```

Comment:
Pour info, lors la suspension d'un compte, on réinitialise son mot de passe : https://github.com/pass-culture/pass-culture-api/blob/89482d1c71dc89d288e1da723916f1cc29ec3f76/src/pcapi/core/users/api.py#L432

Peut-être qu'ici on pourrait préciser quelque chose pour indiquer à l'opérateur que l'utilisateur (dont a réactivé le compte) ne pourra pas utiliser son mot de passe initial. Quelque chose comme :

```suggestion
        flash("Les comptes utilisateurs ont été réactivés. Leur mot de passe a été réinitialisé.")
```

C'est juste une suggestion, à voir si c'est pertinent ou pas. :)